tests/fixture-ts/tests/acceptance/welcome-page-test.js

```diff
@@ -0,0 +1,14 @@
+import { module, test } from 'qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from '<%= name %>/tests/helpers';
+
+module('Acceptance | welcome page', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('visiting /index shows the welcome page', async function (assert) {
+    await visit('/');
+
+    assert.strictEqual(currentURL(), '/');
+    assert.dom('h1').containsText('Congratulations, you made it!');
+  });
+});
```

Comment:
I am commenting on this file but my question is really about all of the fixture-ts folder. I don't see any difference between these files and the JS ones 🤔 do we need to have different input files? and if there is only one or two files difference can we just use the layering/override approach on this so that we only need to specify the differences

Reply:
they'll eventually all be different because they'll be converted to TS, and then help improve the robustness of the glint / lint:types check